### PR TITLE
Update README with gCloud

### PR DESCRIPTION
**Pull request by manitgupta**
Update README with HB on gCloud Information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,6 +110,27 @@ variable if it is not already configured, and
 
 ### Installing HarbourBridge
 
+#### HarbourBridge on gCloud
+You can directly run HarbourBridge from the gCloud CLI instead of building it from source. In order to start using HarbourBridge via Gcloud, the user can install the harbourbridge component of gcloud by executing the below command:
```
Comment (bharadwaj-aditya) [inline README.md:114]: Link this to the gcloud installation instructions.
Also mention this is currently possible only on Linux

**Review by bharadwaj-aditya — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (manitgupta) [reply README.md]: Done

**Review by bharadwaj-aditya — APPROVED** [review]
LGTM

**Comment by bharadwaj-aditya** [comment]
Please also update webv2/README.md

**Pull request merged**